src/main/java/wolox/training/models/User.java

```diff
@@ -58,6 +58,10 @@
     @ApiModelProperty(notes = "The user books: books associated to user")
     private List<Book> books = new ArrayList<>();
 
+    @Column(nullable = false)
+    @ApiModelProperty(notes = "The user password: password of user")
```

Comment:
Here it must indicate that the property is write only. Thus, is not include in the serialization
```suggestion
    @Column(nullable = false)
    @JsonProperty(access = Access.WRITE_ONLY)
    @ApiModelProperty(notes = "The user password: password of user")
```